frontend.py

```diff
@@ -8,10 +8,13 @@
 cors = CORS(app)
 app.config['CORS_HEADERS'] = 'Content-Type'
 
-@app.route('/')
+@app.route('/', methods=['GET'])
 @cross_origin()
 def my_form():
-    return render_template("form.html")
+    username = request.args.get('username', None)
+    if username is None:
```

Comment:
good job @seeeturtle!
Small nitpick: I would have rather used an empty string in the default case rather than using ``None``. Can you do the same?

Reply:
ok.

Reply:
done! :smile: